### PR TITLE
UML-1305: Show trust corporation in attorney list in the summary pages

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose
https://opgtransform.atlassian.net/browse/UML-1305

Fixes UML-1305

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [x] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] New event_codes have been documented on the [wiki page](https://opgtransform.atlassian.net/wiki/spaces/LSML2/pages/3277881441/Understanding+the+event+logs)
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [x] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] I have notified the Interaction Designer of any content changes so that appropriate screenshots/flow diagram changes can be made
* [x] The product team have tested these changes

### CODE REVIEW

File: service-front/app/languages/cy/LC_MESSAGES/messages.html [hunk 1]
```diff
@@ -0,0 +1,6866 @@
+<!DOCTYPE html>
```
Comment (lukejolliffe) [inline service-front/app/languages/cy/LC_MESSAGES/messages.html:1]: Whats this file?

Comment (lukejolliffe) [inline service-front/app/languages/cy/LC_MESSAGES/messages.html:1]: Whats this file?

File: service-front/app/src/Common/src/Service/Lpa/LpaService.php [hunk 2]
```diff
@@ -220,4 +220,28 @@ public function getLpaByCode(string $shareCode, string $donorSurname, ?string $o
 
         return $lpaData;
     }
+
+    /**
+     * @param array $attorneys
+     * @param array $trustCorporations
+     * @return ArrayObject|null
+     */
+    public function getAttorneyDetails(array $attorneys, array $trustCorporations): ?array
```
Comment (lukejolliffe) [inline service-front/app/src/Common/src/Service/Lpa/LpaService.php:229]: Is it a needed requirement to combine the two lists? I would have just read from them both separately in the twig file instead?

**Review by lukejolliffe — CHANGES_REQUESTED** [review]
I think there should also be a BDD test for the UI showing trust corporations? 

Maybe something along the lines of : 

**Given** I have an LPA added with a TrustCorporation as an attorney
**When** I request to view the LPA 
**Then** I can see the trust corporation in the list of attorneys 

?

File: service-front/app/languages/cy/LC_MESSAGES/messages.html (hunk 1 above)
Comment (SeemaMenon) [reply service-front/app/languages/cy/LC_MESSAGES/messages.html]: Deleted file.

File: service-front/app/src/Common/src/Service/Lpa/LpaService.php (hunk 2 above)
Comment (SeemaMenon) [reply service-front/app/src/Common/src/Service/Lpa/LpaService.php:229]: Shouldn't have...but I could not get it read in twig with the correct ordinal sequence.  Let's discuss options tomorrow.

**Comment by SeemaMenon** [comment]
> I think there should also be a BDD test for the UI showing trust corporations?
> 
> Maybe something along the lines of :
> 
> **Given** I have an LPA added with a TrustCorporation as an attorney **When** I request to view the LPA **Then** I can see the trust corporation in the list of attorneys
> 
> ?

@lukejolliffe  - Done

**Comment by andrewpearce-digital** [comment]
@SeemaMenon There's a missing update for the mock integrations nginx.conf file to map the new mock data you've added. This results in the mock returning a 404 for 700000151451. You've likely got it locally, or you can recreate it with the generate_example.sh script.

**Review by lukejolliffe — DISMISSED** [review]

**Review by lukejolliffe — CHANGES_REQUESTED** [review]
The messages.html is back 😅

**Comment by SeemaMenon** [comment]
> The messages.html is back 😅

Removed now.

**Review by lukejolliffe — APPROVED** [review]

**Pull request merged**